client/state/themes/actions.js

```diff
@@ -460,9 +502,10 @@ export function installAndTryAndCustomize( themeId, siteId ) {
  * @param  {String}   siteId       Jetpack Site ID
  * @return {Function}              Action thunk
  */
-export function tryAndCustomize( themeId, siteId ) {
+export function tryAndCustomizeTheme( themeId, siteId ) {
 	return ( dispatch, getState ) => {
-		const theme = getTheme( getState(), siteId, themeId );
+		const siteIdOrWpcom = isJetpackSite( getState(), siteId ) ? siteId : 'wpcom';
```

Comment:
I'd rather not -- Here's it just an impl detail, since right now we need the `theme` object for `getThemeCustomizeUrl`, too. If we change that selectors (and all other `getTheme...Url` ones, which I intend) to use `themeId`, we can rewrite as

```js
if ( isJetpackSite( getState(), siteId ) && ! getTheme( getState(), siteId, themeId ) ) {
  	dispatch( {
  		type: THEME_TRY_AND_CUSTOMIZE_FAILURE,
 		themeId,
 		siteId
 	} );
 	return;
}
const url = getThemeCustomizeUrl( getState(), themeId, siteId );
page( url );
```

And if it's an impl detail, I'd say that indicates we don't need to necessarily move the check to `getTheme`. OTOH, that points at `isJetpackSite( getState(), siteId ) && ! getTheme( getState(), siteId, themeId )` being a recurring pattern, so maybe we should write a selector for that...